### PR TITLE
Resolves: OKTA-231059 - add network and app condition to enroll policy

### PR DESCRIPTION
**Pull request by antonioruberto-okta**
## Description:
- **What's changed?**
  - As part of 2019.10.0 release, the App Condition will be available to be used in the MFA Enroll Policy
  - Also the Network condition has been available for use in MFA Enroll Policy for a long time but network condition link was missing
- **Is this PR related to a Monolith release?**
  - 2019.10.0

How doc looks with changes in this PR
image

### Resolves:

* [OKTA-231059](https://oktainc.atlassian.net/browse/OKTA-231059)

### CODE REVIEW

**Comment by AjayThomas-okta** [comment]
LGTM

Do we want to enhance the App Condition object docs to say that if include and exclude arrays are empty, it means all apps are included to be backwards compatible?

**Comment by antonioruberto-okta** [comment]
@AjayThomas-okta The `Application and App Instance Condition` was added originally for the IdP Discovery Policy. Also it doesn't even mention the `exclude` attribute. It works same for the IdP Policy as well correct?

**Comment by AjayThomas-okta** [comment]
Yes, it is true for IDP routing rules as well (just confirmed by creating an IDP routing rule).
Yup, enhancing the already existing example to have an appId in `exclude` array will be good.
And a second example, to show an empty include/exclude list means all apps included.

**Comment by antonioruberto-okta** [comment]
@AjayThomas-okta Added exclude and added a note

image

Let me know if it makes sense.

**Comment by AjayThomas-okta** [comment]
@vincentvoong-okta pls review note

**Comment by vincentvoong-okta** [comment]
@AjayThomas-okta 

Looks good.

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -962,6 +962,9 @@ Specifies either a general application or specific app instance to match on.
 | Parameter | Description                                           | Data Type | Required |
 | ---       | ---                                                   | ---       | ---      |
 | include   | The list of applications or app instances to match on | Array     | Yes      |
+| exclude   | The list of applications to exclude                   | Array     | Yes      |
+
+> **NOTE:** If both include and exclude are empty, then the condition is met for all applications.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md]: These should use code formmating: If both `include` and `exclude` are empty

**Pull request merged**